### PR TITLE
Feature/UI edge cases

### PR DESCRIPTION
**Pull request by antontsitovich**

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -76,7 +78,26 @@ function App() {
...
         {!error && isLoading && isAddressValid && <Box sx={{ marginTop: 4 }}>Loading...</Box>}
-        {!!error && <Box sx={{ marginTop: 4 }}>Error {`${error}`}</Box>}
+        {!!error && (
+          <Box mt={4}>
+            <CompilationNotification
+              type={NotificationType.NOTIFICATION}
+              title={
+                <NotificationTitle>
+                  <span style={{ color: "#FC5656" }}>Error: </span>
+                  Blockchain error
```
Comment (shaharyakir) [inline src/App.tsx:88]:
```suggestion
                  Unable to fetch contract data
```

File: src/App.tsx [hunk 2]
```diff
@@ -76,7 +78,26 @@ function App() {
       <TopBar />
       <ContentBox>
         {!error && isLoading && isAddressValid && <Box sx={{ marginTop: 4 }}>Loading...</Box>}
-        {!!error && <Box sx={{ marginTop: 4 }}>Error {`${error}`}</Box>}
+        {!!error && (
+          <Box mt={4}>
+            <CompilationNotification
```
Comment (shaharyakir) [inline src/App.tsx:83]: if we're reusing this, it should be named ErrorNotification or something similar

File: src/App.tsx (hunk 1 above)
Comment (antongotech) [reply src/App.tsx]: replaced

File: src/components/AppNotification.tsx [hunk 3]
```diff
@@ -5,7 +5,7 @@ export enum NotificationType {
   ERROR = "Error",
   HINT = "Hint",
   SUCCESS = "Success",
-  NOTIFICATION = "Notification",
+  INFO = "INFO",
```
Comment (shaharyakir) [inline src/components/AppNotification.tsx:8]: INFO = "Info"

**Pull request merged**